File: Day4/checkOddEven.js
// Program to check number is even or odd number..

function checkOddEven(num){

        // All even number is divisible by 2 
        
        if(num % 2 == 0){    
            console.log(num ,"is a even number")
        }

        else{
            console.log(num ,"is a odd number");
        }
    }
    
let num = parseInt(prompt("Enter the number:"));
checkOddEven(num);

File: Day4/CheckOddEven.java
package Day4;
import java.util.Scanner;
public class CheckOddEven {
    public static void checkOddEven(int num){

        if(num % 2== 0){
           System.out.println( num + " is even number.");
        }
        else{
            System.out.println(num + " is odd number");
        }
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the number:");
        int num = sc.nextInt();
        sc.close();
        checkOddEven(num);
    }
}
